Formula/resty.rb

```diff
@@ -1,29 +1,57 @@
 class Resty < Formula
   desc "Command-line REST client that can be used in pipelines"
   homepage "https://github.com/micha/resty"
-  url "https://github.com/micha/resty/archive/2.2.tar.gz"
-  sha256 "a6961e16f1489ffa25bf9f2d8b38b9599c008d49267e32adf93f4f87184d4857"
+  url "https://github.com/micha/resty/archive/2.3.tar.gz"
+  sha256 "65c7bfe1268669d9da5456686c55f1976380b867db5bf12d3603972d7b2d65fd"
   head "https://github.com/micha/resty.git"
 
-  bottle :unneeded
-
   # Don't take +x off these files
```

Comment:
you can add a note that this should be removed for > 2.3

Reply:
just did it